build(deps): weekly dependabot updates

mschilling opened the pull request.

dependabot[bot] committed:
build(deps): bump graphql from 15.8.0 to 16.6.0

Bumps [graphql](https://github.com/graphql/graphql-js) from 15.8.0 to 16.6.0.
- [Release notes](https://github.com/graphql/graphql-js/releases)
- [Commits](https://github.com/graphql/graphql-js/compare/v15.8.0...v16.6.0)

---
updated-dependencies:
- dependency-name: graphql
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump jest-environment-jsdom from 28.1.1 to 29.1.2

Bumps [jest-environment-jsdom](https://github.com/facebook/jest/tree/HEAD/packages/jest-environment-jsdom) from 28.1.1 to 29.1.2.
- [Release notes](https://github.com/facebook/jest/releases)
- [Changelog](https://github.com/facebook/jest/blob/main/CHANGELOG.md)
- [Commits](https://github.com/facebook/jest/commits/v29.1.2/packages/jest-environment-jsdom)

---
updated-dependencies:
- dependency-name: jest-environment-jsdom
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @angular-eslint/eslint-plugin-template

Bumps [@angular-eslint/eslint-plugin-template](https://github.com/angular-eslint/angular-eslint/tree/HEAD/packages/eslint-plugin-template) from 14.0.4 to 14.1.2.
- [Release notes](https://github.com/angular-eslint/angular-eslint/releases)
- [Changelog](https://github.com/angular-eslint/angular-eslint/blob/main/packages/eslint-plugin-template/CHANGELOG.md)
- [Commits](https://github.com/angular-eslint/angular-eslint/commits/v14.1.2/packages/eslint-plugin-template)

---
updated-dependencies:
- dependency-name: "@angular-eslint/eslint-plugin-template"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump apollo-server-express from 3.10.2 to 3.10.3

Bumps [apollo-server-express](https://github.com/apollographql/apollo-server/tree/HEAD/packages/apollo-server-express) from 3.10.2 to 3.10.3.
- [Release notes](https://github.com/apollographql/apollo-server/releases)
- [Changelog](https://github.com/apollographql/apollo-server/blob/main/CHANGELOG.md)
- [Commits](https://github.com/apollographql/apollo-server/commits/apollo-server-express@3.10.3/packages/apollo-server-express)

---
updated-dependencies:
- dependency-name: apollo-server-express
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump svelte-check from 2.9.1 to 2.9.2

Bumps [svelte-check](https://github.com/sveltejs/language-tools) from 2.9.1 to 2.9.2.
- [Release notes](https://github.com/sveltejs/language-tools/releases)
- [Commits](https://github.com/sveltejs/language-tools/compare/svelte-check-2.9.1...svelte-check-2.9.2)

---
updated-dependencies:
- dependency-name: svelte-check
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @typescript-eslint/parser from 5.39.0 to 5.40.0

Bumps [@typescript-eslint/parser](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/parser) from 5.39.0 to 5.40.0.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/parser/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v5.40.0/packages/parser)

---
updated-dependencies:
- dependency-name: "@typescript-eslint/parser"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @babel/preset-env from 7.19.3 to 7.19.4

Bumps [@babel/preset-env](https://github.com/babel/babel/tree/HEAD/packages/babel-preset-env) from 7.19.3 to 7.19.4.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.19.4/packages/babel-preset-env)

---
updated-dependencies:
- dependency-name: "@babel/preset-env"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump eslint-plugin-react from 7.31.8 to 7.31.10

Bumps [eslint-plugin-react](https://github.com/jsx-eslint/eslint-plugin-react) from 7.31.8 to 7.31.10.
- [Release notes](https://github.com/jsx-eslint/eslint-plugin-react/releases)
- [Changelog](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/CHANGELOG.md)
- [Commits](https://github.com/jsx-eslint/eslint-plugin-react/compare/v7.31.8...v7.31.10)

---
updated-dependencies:
- dependency-name: eslint-plugin-react
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump vitest from 0.23.4 to 0.24.1

Bumps [vitest](https://github.com/vitest-dev/vitest) from 0.23.4 to 0.24.1.
- [Release notes](https://github.com/vitest-dev/vitest/releases)
- [Commits](https://github.com/vitest-dev/vitest/compare/v0.23.4...v0.24.1)

---
updated-dependencies:
- dependency-name: vitest
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump postcss from 8.4.17 to 8.4.18

Bumps [postcss](https://github.com/postcss/postcss) from 8.4.17 to 8.4.18.
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/8.4.17...8.4.18)

---
updated-dependencies:
- dependency-name: postcss
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @angular-devkit/build-angular

Bumps [@angular-devkit/build-angular](https://github.com/angular/angular-cli) from 14.2.5 to 14.2.6.
- [Release notes](https://github.com/angular/angular-cli/releases)
- [Changelog](https://github.com/angular/angular-cli/blob/main/CHANGELOG.md)
- [Commits](https://github.com/angular/angular-cli/compare/14.2.5...14.2.6)

---
updated-dependencies:
- dependency-name: "@angular-devkit/build-angular"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

mschilling committed:
Merge pull request #1465 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/angular-devkit/build-angular-14.2.6

build(deps-dev): bump @angular-devkit/build-angular from 14.2.5 to 14.2.6

mschilling committed:
Merge pull request #1447 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/jest-environment-jsdom-29.1.2

build(deps-dev): bump jest-environment-jsdom from 28.1.1 to 29.1.2

mschilling committed:
Merge pull request #1446 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/graphql-16.6.0

build(deps): bump graphql from 15.8.0 to 16.6.0

mschilling committed:
Merge pull request #1450 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/angular-eslint/eslint-plugin-template-14.1.2

build(deps-dev): bump @angular-eslint/eslint-plugin-template from 14.0.4 to 14.1.2

mschilling committed:
Merge pull request #1452 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/apollo-server-express-3.10.3

build(deps): bump apollo-server-express from 3.10.2 to 3.10.3

mschilling committed:
Merge pull request #1454 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/svelte-check-2.9.2

build(deps-dev): bump svelte-check from 2.9.1 to 2.9.2

mschilling committed:
Merge pull request #1455 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/typescript-eslint/parser-5.40.0

build(deps-dev): bump @typescript-eslint/parser from 5.39.0 to 5.40.0

mschilling committed:
Merge pull request #1459 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/babel/preset-env-7.19.4

build(deps-dev): bump @babel/preset-env from 7.19.3 to 7.19.4

mschilling committed:
Merge pull request #1463 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/postcss-8.4.18

build(deps-dev): bump postcss from 8.4.17 to 8.4.18

mschilling committed:
Merge pull request #1462 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/vitest-0.24.1

build(deps-dev): bump vitest from 0.23.4 to 0.24.1

mschilling committed:
Merge pull request #1460 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/eslint-plugin-react-7.31.10

build(deps-dev): bump eslint-plugin-react from 7.31.8 to 7.31.10

dependabot[bot] committed:
build(deps-dev): bump @astrojs/tailwind from 2.0.2 to 2.1.1

Bumps [@astrojs/tailwind](https://github.com/withastro/astro/tree/HEAD/packages/integrations/tailwind) from 2.0.2 to 2.1.1.
- [Release notes](https://github.com/withastro/astro/releases)
- [Changelog](https://github.com/withastro/astro/blob/main/packages/integrations/tailwind/CHANGELOG.md)
- [Commits](https://github.com/withastro/astro/commits/@astrojs/tailwind@2.1.1/packages/integrations/tailwind)

---
updated-dependencies:
- dependency-name: "@astrojs/tailwind"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump cache-manager from 5.0.0 to 5.1.1

Bumps [cache-manager](https://github.com/node-cache-manager/node-cache-manager) from 5.0.0 to 5.1.1.
- [Release notes](https://github.com/node-cache-manager/node-cache-manager/releases)
- [Changelog](https://github.com/node-cache-manager/node-cache-manager/blob/master/History.md)
- [Commits](https://github.com/node-cache-manager/node-cache-manager/compare/5.0.0...5.1.1)

---
updated-dependencies:
- dependency-name: cache-manager
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump tailwindcss from 3.1.8 to 3.2.1

Bumps [tailwindcss](https://github.com/tailwindlabs/tailwindcss) from 3.1.8 to 3.2.1.
- [Release notes](https://github.com/tailwindlabs/tailwindcss/releases)
- [Changelog](https://github.com/tailwindlabs/tailwindcss/blob/master/CHANGELOG.md)
- [Commits](https://github.com/tailwindlabs/tailwindcss/compare/v3.1.8...v3.2.1)

---
updated-dependencies:
- dependency-name: tailwindcss
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump jest-environment-jsdom from 29.1.2 to 29.2.2

Bumps [jest-environment-jsdom](https://github.com/facebook/jest/tree/HEAD/packages/jest-environment-jsdom) from 29.1.2 to 29.2.2.
- [Release notes](https://github.com/facebook/jest/releases)
- [Changelog](https://github.com/facebook/jest/blob/main/CHANGELOG.md)
- [Commits](https://github.com/facebook/jest/commits/v29.2.2/packages/jest-environment-jsdom)

---
updated-dependencies:
- dependency-name: jest-environment-jsdom
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @sveltejs/vite-plugin-svelte from 1.0.9 to 1.1.0

Bumps [@sveltejs/vite-plugin-svelte](https://github.com/sveltejs/vite-plugin-svelte/tree/HEAD/packages/vite-plugin-svelte) from 1.0.9 to 1.1.0.
- [Release notes](https://github.com/sveltejs/vite-plugin-svelte/releases)
- [Changelog](https://github.com/sveltejs/vite-plugin-svelte/blob/main/packages/vite-plugin-svelte/CHANGELOG.md)
- [Commits](https://github.com/sveltejs/vite-plugin-svelte/commits/@sveltejs/vite-plugin-svelte@1.1.0/packages/vite-plugin-svelte)

---
updated-dependencies:
- dependency-name: "@sveltejs/vite-plugin-svelte"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @types/react from 18.0.20 to 18.0.23

Bumps [@types/react](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/react) from 18.0.20 to 18.0.23.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/react)

---
updated-dependencies:
- dependency-name: "@types/react"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump astro from 1.4.5 to 1.5.3

Bumps [astro](https://github.com/withastro/astro/tree/HEAD/packages/astro) from 1.4.5 to 1.5.3.
- [Release notes](https://github.com/withastro/astro/releases)
- [Changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
- [Commits](https://github.com/withastro/astro/commits/astro@1.5.3/packages/astro)

---
updated-dependencies:
- dependency-name: astro
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

mschilling committed:
Merge pull request #1502 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/astro-1.5.3

build(deps-dev): bump astro from 1.4.5 to 1.5.3

mschilling committed:
Merge pull request #1500 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/types/react-18.0.23

build(deps-dev): bump @types/react from 18.0.20 to 18.0.23

mschilling committed:
Merge pull request #1488 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/tailwindcss-3.2.1

build(deps-dev): bump tailwindcss from 3.1.8 to 3.2.1

mschilling committed:
Merge pull request #1483 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/cache-manager-5.1.1

build(deps): bump cache-manager from 5.0.0 to 5.1.1

mschilling committed:
Merge pull request #1481 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/astrojs/tailwind-2.1.1

build(deps-dev): bump @astrojs/tailwind from 2.0.2 to 2.1.1

mschilling committed:
Merge pull request #1499 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/sveltejs/vite-plugin-svelte-1.1.0

build(deps-dev): bump @sveltejs/vite-plugin-svelte from 1.0.9 to 1.1.0

mschilling committed:
Merge pull request #1497 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/jest-environment-jsdom-29.2.2

build(deps-dev): bump jest-environment-jsdom from 29.1.2 to 29.2.2

dependabot[bot] committed:
build(deps-dev): bump autoprefixer from 10.4.12 to 10.4.13 (#1507)

Bumps [autoprefixer](https://github.com/postcss/autoprefixer) from 10.4.12 to 10.4.13.
- [Release notes](https://github.com/postcss/autoprefixer/releases)
- [Changelog](https://github.com/postcss/autoprefixer/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/autoprefixer/compare/10.4.12...10.4.13)

---
updated-dependencies:
- dependency-name: autoprefixer
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump @types/node from 18.8.3 to 18.11.7 (#1501)

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 18.8.3 to 18.11.7.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump regenerator-runtime from 0.13.9 to 0.13.10 (#1468)

Bumps [regenerator-runtime](https://github.com/facebook/regenerator) from 0.13.9 to 0.13.10.
- [Release notes](https://github.com/facebook/regenerator/releases)
- [Commits](https://github.com/facebook/regenerator/compare/regenerator-runtime@0.13.9...regenerator-runtime@0.13.10)

---
updated-dependencies:
- dependency-name: regenerator-runtime
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump vitest from 0.24.1 to 0.24.3 (#1471)

Bumps [vitest](https://github.com/vitest-dev/vitest) from 0.24.1 to 0.24.3.
- [Release notes](https://github.com/vitest-dev/vitest/releases)
- [Commits](https://github.com/vitest-dev/vitest/compare/v0.24.1...v0.24.3)

---
updated-dependencies:
- dependency-name: vitest
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump eslint from 8.24.0 to 8.26.0 (#1489)

Bumps [eslint](https://github.com/eslint/eslint) from 8.24.0 to 8.26.0.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Changelog](https://github.com/eslint/eslint/blob/main/CHANGELOG.md)
- [Commits](https://github.com/eslint/eslint/compare/v8.24.0...v8.26.0)

---
updated-dependencies:
- dependency-name: eslint
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump @angular-eslint/eslint-plugin (#1444)

Bumps [@angular-eslint/eslint-plugin](https://github.com/angular-eslint/angular-eslint/tree/HEAD/packages/eslint-plugin) from 14.0.4 to 14.1.2.
- [Release notes](https://github.com/angular-eslint/angular-eslint/releases)
- [Changelog](https://github.com/angular-eslint/angular-eslint/blob/main/packages/eslint-plugin/CHANGELOG.md)
- [Commits](https://github.com/angular-eslint/angular-eslint/commits/v14.1.2/packages/eslint-plugin)

---
updated-dependencies:
- dependency-name: "@angular-eslint/eslint-plugin"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump @babel/core from 7.19.3 to 7.19.6 (#1486)

Bumps [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) from 7.19.3 to 7.19.6.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.19.6/packages/babel-core)

---
updated-dependencies:
- dependency-name: "@babel/core"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump svelte from 3.50.1 to 3.52.0 (#1474)

Bumps [svelte](https://github.com/sveltejs/svelte) from 3.50.1 to 3.52.0.
- [Release notes](https://github.com/sveltejs/svelte/releases)
- [Changelog](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sveltejs/svelte/compare/v3.50.1...v3.52.0)

---
updated-dependencies:
- dependency-name: svelte
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump @types/luxon from 3.0.1 to 3.0.2 (#1475)

Bumps [@types/luxon](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/luxon) from 3.0.1 to 3.0.2.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/luxon)

---
updated-dependencies:
- dependency-name: "@types/luxon"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump astro from 1.5.3 to 1.6.0 (#1506)

Bumps [astro](https://github.com/withastro/astro/tree/HEAD/packages/astro) from 1.5.3 to 1.6.0.
- [Release notes](https://github.com/withastro/astro/releases)
- [Changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
- [Commits](https://github.com/withastro/astro/commits/astro@1.6.0/packages/astro)

---
updated-dependencies:
- dependency-name: astro
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump @types/react from 18.0.23 to 18.0.24 (#1504)

Bumps [@types/react](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/react) from 18.0.23 to 18.0.24.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/react)

---
updated-dependencies:
- dependency-name: "@types/react"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump @typescript-eslint/eslint-plugin (#1495)

Bumps [@typescript-eslint/eslint-plugin](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/eslint-plugin) from 5.39.0 to 5.41.0.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v5.41.0/packages/eslint-plugin)

---
updated-dependencies:
- dependency-name: "@typescript-eslint/eslint-plugin"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump @angular-devkit/build-angular (#1505)

Bumps [@angular-devkit/build-angular](https://github.com/angular/angular-cli) from 14.2.6 to 14.2.7.
- [Release notes](https://github.com/angular/angular-cli/releases)
- [Changelog](https://github.com/angular/angular-cli/blob/main/CHANGELOG.md)
- [Commits](https://github.com/angular/angular-cli/compare/14.2.6...14.2.7)

---
updated-dependencies:
- dependency-name: "@angular-devkit/build-angular"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump core-js from 3.25.5 to 3.26.0 (#1490)

Bumps [core-js](https://github.com/zloirock/core-js/tree/HEAD/packages/core-js) from 3.25.5 to 3.26.0.
- [Release notes](https://github.com/zloirock/core-js/releases)
- [Changelog](https://github.com/zloirock/core-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/zloirock/core-js/commits/v3.26.0/packages/core-js)

---
updated-dependencies:
- dependency-name: core-js
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump @typescript-eslint/parser from 5.40.0 to 5.41.0 (#1494)

Bumps [@typescript-eslint/parser](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/parser) from 5.40.0 to 5.41.0.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/parser/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v5.41.0/packages/parser)

---
updated-dependencies:
- dependency-name: "@typescript-eslint/parser"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump @types/react-dom from 18.0.6 to 18.0.8 (#1503)

Bumps [@types/react-dom](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/react-dom) from 18.0.6 to 18.0.8.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/react-dom)

---
updated-dependencies:
- dependency-name: "@types/react-dom"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump @commitlint/cli from 17.1.2 to 17.2.0

Bumps [@commitlint/cli](https://github.com/conventional-changelog/commitlint/tree/HEAD/@commitlint/cli) from 17.1.2 to 17.2.0.
- [Release notes](https://github.com/conventional-changelog/commitlint/releases)
- [Changelog](https://github.com/conventional-changelog/commitlint/blob/master/@commitlint/cli/CHANGELOG.md)
- [Commits](https://github.com/conventional-changelog/commitlint/commits/v17.2.0/@commitlint/cli)

---
updated-dependencies:
- dependency-name: "@commitlint/cli"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump tslib from 2.4.0 to 2.4.1

Bumps [tslib](https://github.com/Microsoft/tslib) from 2.4.0 to 2.4.1.
- [Release notes](https://github.com/Microsoft/tslib/releases)
- [Commits](https://github.com/Microsoft/tslib/compare/2.4.0...2.4.1)

---
updated-dependencies:
- dependency-name: tslib
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump astro from 1.6.0 to 1.6.2

Bumps [astro](https://github.com/withastro/astro/tree/HEAD/packages/astro) from 1.6.0 to 1.6.2.
- [Release notes](https://github.com/withastro/astro/releases)
- [Changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
- [Commits](https://github.com/withastro/astro/commits/astro@1.6.2/packages/astro)

---
updated-dependencies:
- dependency-name: astro
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump vitest from 0.24.3 to 0.24.5

Bumps [vitest](https://github.com/vitest-dev/vitest/tree/HEAD/packages/vitest) from 0.24.3 to 0.24.5.
- [Release notes](https://github.com/vitest-dev/vitest/releases)
- [Commits](https://github.com/vitest-dev/vitest/commits/v0.24.5/packages/vitest)

---
updated-dependencies:
- dependency-name: vitest
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @types/node from 18.11.7 to 18.11.9

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 18.11.7 to 18.11.9.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump react-router-dom from 6.4.2 to 6.4.3

Bumps [react-router-dom](https://github.com/remix-run/react-router/tree/HEAD/packages/react-router-dom) from 6.4.2 to 6.4.3.
- [Release notes](https://github.com/remix-run/react-router/releases)
- [Changelog](https://github.com/remix-run/react-router/blob/main/packages/react-router-dom/CHANGELOG.md)
- [Commits](https://github.com/remix-run/react-router/commits/react-router-dom@6.4.3/packages/react-router-dom)

---
updated-dependencies:
- dependency-name: react-router-dom
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump apollo-server-express from 3.10.3 to 3.11.1

Bumps [apollo-server-express](https://github.com/apollographql/apollo-server/tree/HEAD/packages/apollo-server-express) from 3.10.3 to 3.11.1.
- [Release notes](https://github.com/apollographql/apollo-server/releases)
- [Changelog](https://github.com/apollographql/apollo-server/blob/apollo-server-express@3.11.1/CHANGELOG.md)
- [Commits](https://github.com/apollographql/apollo-server/commits/apollo-server-express@3.11.1/packages/apollo-server-express)

---
updated-dependencies:
- dependency-name: apollo-server-express
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @angular-devkit/build-angular

Bumps [@angular-devkit/build-angular](https://github.com/angular/angular-cli) from 14.2.7 to 14.2.8.
- [Release notes](https://github.com/angular/angular-cli/releases)
- [Changelog](https://github.com/angular/angular-cli/blob/main/CHANGELOG.md)
- [Commits](https://github.com/angular/angular-cli/compare/14.2.7...14.2.8)

---
updated-dependencies:
- dependency-name: "@angular-devkit/build-angular"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

mschilling committed:
Merge pull request #1526 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/angular-devkit/build-angular-14.2.8

build(deps-dev): bump @angular-devkit/build-angular from 14.2.7 to 14.2.8

mschilling committed:
Merge pull request #1525 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/apollo-server-express-3.11.1

build(deps): bump apollo-server-express from 3.10.3 to 3.11.1

mschilling committed:
Merge pull request #1523 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/react-router-dom-6.4.3

build(deps): bump react-router-dom from 6.4.2 to 6.4.3

mschilling committed:
Merge pull request #1522 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/types/node-18.11.9

build(deps-dev): bump @types/node from 18.11.7 to 18.11.9

mschilling committed:
Merge pull request #1520 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/vitest-0.24.5

build(deps-dev): bump vitest from 0.24.3 to 0.24.5

mschilling committed:
Merge pull request #1519 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/astro-1.6.2

build(deps-dev): bump astro from 1.6.0 to 1.6.2

mschilling committed:
Merge pull request #1517 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/tslib-2.4.1

build(deps): bump tslib from 2.4.0 to 2.4.1

mschilling committed:
Merge pull request #1514 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/commitlint/cli-17.2.0

build(deps-dev): bump @commitlint/cli from 17.1.2 to 17.2.0

dependabot[bot] committed:
build(deps): bump cache-manager from 5.1.1 to 5.1.3

Bumps [cache-manager](https://github.com/node-cache-manager/node-cache-manager) from 5.1.1 to 5.1.3.
- [Release notes](https://github.com/node-cache-manager/node-cache-manager/releases)
- [Changelog](https://github.com/node-cache-manager/node-cache-manager/blob/master/CHANGELOG.md)
- [Commits](https://github.com/node-cache-manager/node-cache-manager/compare/5.1.1...5.1.3)

---
updated-dependencies:
- dependency-name: cache-manager
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump luxon from 3.0.4 to 3.1.0

Bumps [luxon](https://github.com/moment/luxon) from 3.0.4 to 3.1.0.
- [Release notes](https://github.com/moment/luxon/releases)
- [Changelog](https://github.com/moment/luxon/blob/master/CHANGELOG.md)
- [Commits](https://github.com/moment/luxon/compare/3.0.4...3.1.0)

---
updated-dependencies:
- dependency-name: luxon
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @commitlint/config-conventional

Bumps [@commitlint/config-conventional](https://github.com/conventional-changelog/commitlint/tree/HEAD/@commitlint/config-conventional) from 17.1.0 to 17.2.0.
- [Release notes](https://github.com/conventional-changelog/commitlint/releases)
- [Changelog](https://github.com/conventional-changelog/commitlint/blob/master/@commitlint/config-conventional/CHANGELOG.md)
- [Commits](https://github.com/conventional-changelog/commitlint/commits/v17.2.0/@commitlint/config-conventional)

---
updated-dependencies:
- dependency-name: "@commitlint/config-conventional"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

nratering reviewed (approved):
Fix merge conflict ;-)

dependabot[bot] committed:
build(deps-dev): bump @astrojs/tailwind from 2.1.1 to 2.1.2

Bumps [@astrojs/tailwind](https://github.com/withastro/astro/tree/HEAD/packages/integrations/tailwind) from 2.1.1 to 2.1.2.
- [Release notes](https://github.com/withastro/astro/releases)
- [Changelog](https://github.com/withastro/astro/blob/main/packages/integrations/tailwind/CHANGELOG.md)
- [Commits](https://github.com/withastro/astro/commits/@astrojs/tailwind@2.1.2/packages/integrations/tailwind)

---
updated-dependencies:
- dependency-name: "@astrojs/tailwind"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump sass from 1.55.0 to 1.56.1

Bumps [sass](https://github.com/sass/dart-sass) from 1.55.0 to 1.56.1.
- [Release notes](https://github.com/sass/dart-sass/releases)
- [Changelog](https://github.com/sass/dart-sass/blob/main/CHANGELOG.md)
- [Commits](https://github.com/sass/dart-sass/compare/1.55.0...1.56.1)

---
updated-dependencies:
- dependency-name: sass
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump astro from 1.6.2 to 1.6.5

Bumps [astro](https://github.com/withastro/astro/tree/HEAD/packages/astro) from 1.6.2 to 1.6.5.
- [Release notes](https://github.com/withastro/astro/releases)
- [Changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
- [Commits](https://github.com/withastro/astro/commits/astro@1.6.5/packages/astro)

---
updated-dependencies:
- dependency-name: astro
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump jest-environment-jsdom from 29.2.2 to 29.3.1

Bumps [jest-environment-jsdom](https://github.com/facebook/jest/tree/HEAD/packages/jest-environment-jsdom) from 29.2.2 to 29.3.1.
- [Release notes](https://github.com/facebook/jest/releases)
- [Changelog](https://github.com/facebook/jest/blob/main/CHANGELOG.md)
- [Commits](https://github.com/facebook/jest/commits/v29.3.1/packages/jest-environment-jsdom)

---
updated-dependencies:
- dependency-name: jest-environment-jsdom
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump tailwindcss from 3.2.1 to 3.2.2

Bumps [tailwindcss](https://github.com/tailwindlabs/tailwindcss) from 3.2.1 to 3.2.2.
- [Release notes](https://github.com/tailwindlabs/tailwindcss/releases)
- [Changelog](https://github.com/tailwindlabs/tailwindcss/blob/master/CHANGELOG.md)
- [Commits](https://github.com/tailwindlabs/tailwindcss/compare/v3.2.1...v3.2.2)

---
updated-dependencies:
- dependency-name: tailwindcss
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump husky from 8.0.1 to 8.0.2

Bumps [husky](https://github.com/typicode/husky) from 8.0.1 to 8.0.2.
- [Release notes](https://github.com/typicode/husky/releases)
- [Commits](https://github.com/typicode/husky/compare/v8.0.1...v8.0.2)

---
updated-dependencies:
- dependency-name: husky
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @typescript-eslint/eslint-plugin

Bumps [@typescript-eslint/eslint-plugin](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/eslint-plugin) from 5.41.0 to 5.42.1.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v5.42.1/packages/eslint-plugin)

---
updated-dependencies:
- dependency-name: "@typescript-eslint/eslint-plugin"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump eslint from 8.26.0 to 8.27.0

Bumps [eslint](https://github.com/eslint/eslint) from 8.26.0 to 8.27.0.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Changelog](https://github.com/eslint/eslint/blob/main/CHANGELOG.md)
- [Commits](https://github.com/eslint/eslint/compare/v8.26.0...v8.27.0)

---
updated-dependencies:
- dependency-name: eslint
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @types/react from 18.0.24 to 18.0.25

Bumps [@types/react](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/react) from 18.0.24 to 18.0.25.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/react)

---
updated-dependencies:
- dependency-name: "@types/react"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @babel/preset-env from 7.19.4 to 7.20.2

Bumps [@babel/preset-env](https://github.com/babel/babel/tree/HEAD/packages/babel-preset-env) from 7.19.4 to 7.20.2.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.20.2/packages/babel-preset-env)

---
updated-dependencies:
- dependency-name: "@babel/preset-env"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump vitest from 0.24.5 to 0.25.1

Bumps [vitest](https://github.com/vitest-dev/vitest/tree/HEAD/packages/vitest) from 0.24.5 to 0.25.1.
- [Release notes](https://github.com/vitest-dev/vitest/releases)
- [Commits](https://github.com/vitest-dev/vitest/commits/v0.25.1/packages/vitest)

---
updated-dependencies:
- dependency-name: vitest
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

mschilling committed:
Merge pull request #1542 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/vitest-0.25.1

build(deps-dev): bump vitest from 0.24.5 to 0.25.1

mschilling committed:
Merge pull request #1540 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/babel/preset-env-7.20.2

build(deps-dev): bump @babel/preset-env from 7.19.4 to 7.20.2

mschilling committed:
Merge pull request #1538 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/types/react-18.0.25

build(deps-dev): bump @types/react from 18.0.24 to 18.0.25

mschilling committed:
Merge pull request #1537 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/eslint-8.27.0

build(deps-dev): bump eslint from 8.26.0 to 8.27.0

mschilling committed:
Merge pull request #1536 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/typescript-eslint/eslint-plugin-5.42.1

build(deps-dev): bump @typescript-eslint/eslint-plugin from 5.41.0 to 5.42.1

mschilling committed:
Merge pull request #1534 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/husky-8.0.2

build(deps-dev): bump husky from 8.0.1 to 8.0.2

mschilling committed:
Merge pull request #1533 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/tailwindcss-3.2.2

build(deps-dev): bump tailwindcss from 3.2.1 to 3.2.2

mschilling committed:
Merge pull request #1532 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/jest-environment-jsdom-29.3.1

build(deps-dev): bump jest-environment-jsdom from 29.2.2 to 29.3.1

mschilling committed:
Merge pull request #1530 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/sass-1.56.1

build(deps-dev): bump sass from 1.55.0 to 1.56.1

mschilling committed:
Merge pull request #1524 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/cache-manager-5.1.3

build(deps): bump cache-manager from 5.1.1 to 5.1.3

mschilling committed:
Merge pull request #1516 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/luxon-3.1.0

build(deps): bump luxon from 3.0.4 to 3.1.0

mschilling committed:
Merge pull request #1528 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/astrojs/tailwind-2.1.2

build(deps-dev): bump @astrojs/tailwind from 2.1.1 to 2.1.2

mschilling committed:
Merge pull request #1511 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/commitlint/config-conventional-17.2.0

build(deps-dev): bump @commitlint/config-conventional from 17.1.0 to 17.2.0

mschilling committed:
Merge pull request #1531 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/astro-1.6.5

build(deps-dev): bump astro from 1.6.2 to 1.6.5

dependabot[bot] committed:
build(deps-dev): bump @babel/core from 7.19.6 to 7.20.2

Bumps [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) from 7.19.6 to 7.20.2.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.20.2/packages/babel-core)

---
updated-dependencies:
- dependency-name: "@babel/core"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @angular-devkit/build-angular

Bumps [@angular-devkit/build-angular](https://github.com/angular/angular-cli) from 14.2.8 to 14.2.9.
- [Release notes](https://github.com/angular/angular-cli/releases)
- [Changelog](https://github.com/angular/angular-cli/blob/main/CHANGELOG.md)
- [Commits](https://github.com/angular/angular-cli/compare/14.2.8...14.2.9)

---
updated-dependencies:
- dependency-name: "@angular-devkit/build-angular"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @types/luxon from 3.0.2 to 3.1.0

Bumps [@types/luxon](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/luxon) from 3.0.2 to 3.1.0.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/luxon)

---
updated-dependencies:
- dependency-name: "@types/luxon"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump tailwindcss from 3.2.2 to 3.2.3

Bumps [tailwindcss](https://github.com/tailwindlabs/tailwindcss) from 3.2.2 to 3.2.3.
- [Release notes](https://github.com/tailwindlabs/tailwindcss/releases)
- [Changelog](https://github.com/tailwindlabs/tailwindcss/blob/master/CHANGELOG.md)
- [Commits](https://github.com/tailwindlabs/tailwindcss/compare/v3.2.2...v3.2.3)

---
updated-dependencies:
- dependency-name: tailwindcss
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump svelte from 3.52.0 to 3.53.1

Bumps [svelte](https://github.com/sveltejs/svelte) from 3.52.0 to 3.53.1.
- [Release notes](https://github.com/sveltejs/svelte/releases)
- [Changelog](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sveltejs/svelte/compare/v3.52.0...v3.53.1)

---
updated-dependencies:
- dependency-name: svelte
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump astro from 1.6.5 to 1.6.7

Bumps [astro](https://github.com/withastro/astro/tree/HEAD/packages/astro) from 1.6.5 to 1.6.7.
- [Release notes](https://github.com/withastro/astro/releases)
- [Changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
- [Commits](https://github.com/withastro/astro/commits/astro@1.6.7/packages/astro)

---
updated-dependencies:
- dependency-name: astro
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump postcss from 8.4.18 to 8.4.19

Bumps [postcss](https://github.com/postcss/postcss) from 8.4.18 to 8.4.19.
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/8.4.18...8.4.19)

---
updated-dependencies:
- dependency-name: postcss
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

mschilling committed:
Merge pull request #1550 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/postcss-8.4.19

build(deps-dev): bump postcss from 8.4.18 to 8.4.19

mschilling committed:
Merge pull request #1549 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/astro-1.6.7

build(deps-dev): bump astro from 1.6.5 to 1.6.7

mschilling committed:
Merge pull request #1548 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/svelte-3.53.1

build(deps-dev): bump svelte from 3.52.0 to 3.53.1

mschilling committed:
Merge pull request #1546 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/tailwindcss-3.2.3

build(deps-dev): bump tailwindcss from 3.2.2 to 3.2.3

mschilling committed:
Merge pull request #1545 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/types/luxon-3.1.0

build(deps-dev): bump @types/luxon from 3.0.2 to 3.1.0

mschilling committed:
Merge pull request #1544 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/angular-devkit/build-angular-14.2.9

build(deps-dev): bump @angular-devkit/build-angular from 14.2.8 to 14.2.9

mschilling committed:
Merge pull request #1539 from move4mobile/dependabot/npm_and_yarn/integration/dependabot/babel/core-7.20.2

build(deps-dev): bump @babel/core from 7.19.6 to 7.20.2

mschilling committed:
Merge branch 'develop' into integration/dependabot

* develop:
  chore: updated develop, tailwind
  fix: tailwind fixed tailwind and updated astro to version 1 beta
